src/ProgramBlockEditor.scss

```diff
@@ -131,6 +131,10 @@ button.ProgramBlockEditor__program-block {
     margin-right: 1.4rem;
 }
 
+button.ProgramBlockEditor__program-block--disabled {
+    fill: $c2lc-disabled-color;
+}
+
```

Comment:
We don't need this new rule as we want disabled program blocks to look unchanged when disabled.